<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Przebieg_zmiennosci</class>
 <widget class="QWidget" name="Przebieg_zmiennosci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1651</width>
    <height>902</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Przebieg zmienności funkcji</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: Arial;
	font-size: 24px;
}
#centralwidget{
	background-color: #0ca678;
}
QPushButton{
	height: 60px;
	border: 2px solid white;
	border-radius: 12px;
	color: white;
	background-color: transparent;
}
QPushButton:hover{
background-color: white; 
color: #0ca678;
}
QLabel{
	padding-right: 10px;
	border: 2px solid white;
	border-radius: 12px;
	color: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2161</width>
     <height>951</height>
    </rect>
   </property>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>1480</x>
      <y>20</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Zapisz wykres</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>1451</width>
      <height>871</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">border:none</string>
       </property>
       <property name="text">
        <string>1. Dziedzina: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Dziedzina">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">border:none</string>
       </property>
       <property name="text">
        <string>2. Miejsca zerowe:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Zerowe">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">border:none</string>
       </property>
       <property name="text">
        <string>3. Punkt przecięcia z osią OY:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_OY">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">border:none</string>
       </property>
       <property name="text">
        <string>4. Granice:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Granice">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">border:none</string>
       </property>
       <property name="text">
        <string>5. Asymptoty:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Asymptoty">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">border:none</string>
       </property>
       <property name="text">
        <string>6. Przedziały monotoniczności:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Mon">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">border:none</string>
       </property>
       <property name="text">
        <string>7. Ekstrema lokalne:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Ext">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
